app/src/org/commcare/activities/HomeScreenBaseActivity.java

```diff
@@ -1010,13 +1010,14 @@ private void formEntry(Uri formUri, FormRecord r, String headerTitle) {
         startActivityForResult(i, MODEL_RESULT);
     }
 
-    /**
-     * Triggered when an automatic sync is pending
-     */
-    private void handlePendingSync() {
-        long lastSync = CommCareApplication.instance().getCurrentApp().getAppPreferences().getLong("last-ota-restore", 0);
-        String footer = lastSync == 0 ? "never" : SimpleDateFormat.getDateTimeInstance().format(lastSync);
-        Logger.log(LogTypes.TYPE_USER, "autosync triggered. Last Sync|" + footer);
+    private void triggerSync(boolean triggeredByAutoSyncPending) {
+        if (triggeredByAutoSyncPending) {
```

Comment:
Is the thinking here that logging this sync attempt will be redundant when we've just updated (as we know that we'll now always attempt syncing in this event)?

Reply:
No I just did this because the message that's being logged here is about an auto-sync specifically, and supposedly what's meant to be conveyed by this log message is that we're triggering one now, along with when we last triggered one.